objects/src/notes/metadata.rs

```diff
@@ -30,6 +30,9 @@ pub struct NoteMetadata {
 
     /// An arbitrary user-defined value.
     aux: Felt,
+
+    /// Expresses when a note is ready to be consumed.
```

Comment:
nit: I would change "Expresses" to "Specifies".

Reply:
Changed. Was hesitant to make it more assertive at first because the note script can end up specifying different conditions.